sips/sip-submission.md

```diff
@@ -132,8 +132,9 @@ During every iteration, the appointed reviewer presents the changes (updated
 design document, progress with the implementation, etc) to the SIP Committee.
 Based on the feedback, the SIP is either:
 
-1. Accepted, in which case the committee proposes that the compiler maintainers
-   include the new language feature in a future release of the Scala compiler.   
+1. Accepted, in which case the committee hands over to the compiler maintainers
```

Comment:
you can't say "hands over to" without a direct object. you could say "hands the matter over to"